messaging.tests/Integration/BundlesControllerTests.cs

```diff
@@ -557,6 +557,22 @@ public async void PostNCHSIsInDestinationEndpointList()
             Assert.Equal(HttpStatusCode.NoContent, createSubmissionMessage.StatusCode);
         }
 
+        [Fact]
+        public async void PostNCHSIsInDestinationEndpointListUppercase()
+        {
+            // Clear any messages in the database for a clean test
+            DatabaseHelper.ResetDatabase(_context);
+
+            // Create a new empty Death Record WITH nchs in the endpoint list
+            DeathRecordSubmissionMessage recordSubmission = new DeathRecordSubmissionMessage(new DeathRecord());
+            recordSubmission.MessageSource = "http://example.fhir.org";
+            recordSubmission.CertNo = 1;
+            recordSubmission.MessageDestination = "temp,http://nchs.CDC.gov/VRDR_Submission,temp";
```

Comment:
did the "temp," get added when I updated the branch? Its causing this test to fail